addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -35,7 +35,7 @@ Auto-discovery is enabled by default.
 To disable it, you can add the following line to `<openHAB-conf>/services/runtime.cfg`:
 
 ```
-org.openhab.kodi:enableAutoDiscovery=false
+discovery.kodi:background=false
```

Comment:
Is this a documentation change, or really a change of functionality, otherwise we must make sure it ends up in the releasenotes

Reply:
Do we already have a WIKI page for the next release? If not I propose to label this PR as APIBreaking.